src/main/java/uk/gov/companieshouse/overseasentitiesapi/model/SavePoint.java

```diff
@@ -0,0 +1,16 @@
+package uk.gov.companieshouse.overseasentitiesapi.model;
+
+/**
+ * This indicates the point in the overseas entity registration journey that *valid* data is being saved at. Validation
+ * checks run against all the data up to this point must pass for the save to be successful.
+ */
+public enum SavePoint {
+    PRESENTER,
```

Comment:
I know we're not doing validation of presenter data in this story, but put this value in as an example.